### PR TITLE
Metaomics sig webpage

### PR DESCRIPTION
**Pull request by CarsonJM**
Re-opens #2466 and incorporates changes made in that PR

@netlify /special-interest-groups/meta-omics

### CODE REVIEW

**Comment by CarsonJM** [comment]
@jfy133 We're back in action!

File: src/content/special-interest-groups/meta-omics/meetings.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'Meetings'
+subtitle: 'When and where the Meta-omics group meets'
+groupName: Meta-omics
+---
+
+# Meeting schedule
+
```
Comment (mashehu) [inline src/content/special-interest-groups/meta-omics/meetings.md:8]:
```suggestion

Details coming soon!
```

**Review by mashehu — APPROVED** [review]

**Comment by CarsonJM** [comment]
@nf-core-bot Fix linting

File: src/content/special-interest-groups/meta-omics/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+1. helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
+2. implementing semi-'chaining' of sequential pipelines
+3. developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
+4. documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
+
+Depending on interest/activity, other potential activities could include:
+
+1. benchmarking meta-omic pipelines using relevant datasets (i.e. CAMI2), or reads generated using #readsimulator
+2. adding nf-tests to meta-omic pipelines
```
Comment (jfy133) [inline src/content/special-interest-groups/meta-omics/index.md:38]: A bit confused what  this is referring to now :sweat_smile:

File: src/content/special-interest-groups/meta-omics/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+3. reducing functional redundancy between pipelines
+
+# Activities
+
+When were we thinking of having meetings?
+Are these going to be development meetings, or planning meetings?
+What platform should we post meetings on?
+Should we create a project board that we can post tasks on?
```
Comment (jfy133) [inline src/content/special-interest-groups/meta-omics/index.md:46]:
```suggestion
Initial activities will involve a monthly meeting with developers and users, to identify the most relevant problems related meta-omics with nf-core pipelines. Future activities may include presentations and discussions from topic-experts, as well as  tutorials, workshops, and demo-days of nf-core pipelines.
```

**Review by jfy133 — COMMENTED** [review]
@nf-core/core any more thoughts ? Again I don't feel I should be approving my ownstuff :sweat_smile:

File: src/content/special-interest-groups/meta-omics/meetings.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+Details coming soon!
+
+# How to join
```
Comment (ewels) [inline src/content/special-interest-groups/meta-omics/meetings.md:11]: Could always put a link to the Slack group if you want..

```suggestion
```

**Review by ewels — APPROVED** [review]
Nitpicks, but mostly looking fine as a stub 👍🏻

File: src/content/special-interest-groups/meta-omics/index.md (hunk 2 above)
Comment (ewels) [reply src/content/special-interest-groups/meta-omics/index.md]: I leave this one to you two 😅 Otherwise LGTM to merge 👍🏻

**Comment by ewels** [comment]
@nf-core-bot fix linting

**Comment by ewels** [comment]
Merging as I need the URL. Can me improved post-merge.

**Pull request merged**